apps/poker-roll/src/app/bottomNav.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Box from '@mui/material/Box';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import BottomNavigationAction from '@mui/material/BottomNavigationAction';
+import EmojiEventsIcon from '@mui/icons-material/EmojiEvents';
+import TableBarIcon from '@mui/icons-material/TableBar';
+import PaidIcon from '@mui/icons-material/Paid';
+
+export default function BottomNav() {
+    const [value, setValue] = React.useState(0);
+  
+    return (
+        <div className="footer">
+            <Box sx={{ width: 500 }}>
+                <BottomNavigation
+                 showLabels
+                 value={value}
+                    onChange={(event, newValue) => {
+                    setValue(newValue);
+                }}
+            >
+                    <BottomNavigationAction href="/bankroll" label="BankRoll"  icon={<PaidIcon />} />
+                    <BottomNavigationAction href="/cashgame" label="CashGames"  icon={<TableBarIcon />} />
+                    <BottomNavigationAction href="/tournament" label="Tournament"  icon={<EmojiEventsIcon />} />
+                </BottomNavigation>
+             </Box>
+        </div>
+    );
+}
```

Comment:
@AMess33 What you've got here may technically be working but it's got a major downside in that it's not hooked up with react-router. These href props make it function like a multi-page website instead of a single page app. Notice when you click a nav action the entire page refreshes. When the app starts to grow and contains more data those refreshes will start to take longer and become much more noticeable. We want our route transitions to be seamless. Also on refresh, the `value` state gets re-initialized to `0` so your `onChange` handler doesn't really matter. Notice the highlighted nav action never changes since the `value` is always being reset back to `0`.

I'd recommend renaming your `href` props to `value` so each nav action has a value of the path it should point to. Then in your `onChange` handler, `newValue` will be the clicked action's path `value`. From there, find a react-router hook that you can use to navigate to the new path.

The main BottomNavigation wrapper component's `value` prop should align with the selected action `value`. You don't want to use a custom `useState` hook for that, react-router already knows what the current path is. Check out the `useLocation` hook one more time, here's the properties it gives you access to:

```
{
  hash
  key
  pathname
  search
  state
}
```

It might also be helpful to add some placeholder text to your other page components so you can quickly tell when the page content actually changes while you're developing.